conda/models/records.py

```diff
@@ -245,6 +245,9 @@ class PackageRecord(DictSafeMixin, Entity):
 
     md5 = StringField(default=None, required=False, nullable=True, default_in_dump=False)
     url = StringField(default=None, required=False, nullable=True, default_in_dump=False)
+    sha256 = StringField(default=None, required=False, nullable=True, default_in_dump=False)
+    conda_outer_sha256 = StringField(default=None, required=False, nullable=True,
```

Comment:
why does this not get a `conda_inner_sha256` as well?

Reply:
Seems like a mistake.  Will add it in.